src/components/WebComponentProject/WebComponentProject.jsx

```diff
@@ -19,7 +19,7 @@ import {
   stepChangedEvent,
 } from "../../events/WebComponentCustomEvents";
 
-const WebComponentProject = () => {
+const WebComponentProject = ({ withSidebar = false, sidebarOptions }) => {
```

Comment:
might need a default value if you're dependent on array methods later
```suggestion
const WebComponentProject = ({ withSidebar = false, sidebarOptions = [] }) => {
```